content/en/docs/Contribute/_index.md

```diff
@@ -0,0 +1,528 @@
+---
+title: Contribute
+description: Learn how to contribute to the OpenTelemetry documentation.
+weight: 200
+cSpell:ignore: linktitle open-telemetry prepopulated repos spacewhite
+---
+
+You can open an issue about OpenTelemetry documentation, or contribute a change
+with a pull request (PR) to the
+[`opentelemetry.io` GitHub repository](https://github.com/open-telemetry/opentelemetry.io).
+
+OpenTelemetry documentation contributors:
+
+- Improve existing content.
+- Create new content.
+- Update the OpenTelemetry Registry.
+- Improve the code that builds the site.
+
+See also the general
+[OpenTelemetry Contributor Guide](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)
+, which provides details on the Contributor License Agreement and the Code of
+Conduct.
+
+## Requirements
+
+To contribute, you need to be familiar with the following techs and tools:
+
+- git
+- GitHub
+- Markdown (CommonMark)
+- YAML
+
+For technical details concerning how the documentation is built and tested
+locally, see the
+[CONTRIBUTING.md](https://github.com/open-telemetry/opentelemetry.io/blob/main/CONTRIBUTING.md)
+file.
+
+### Sign the CNCF CLA {#sign-the-cla}
+
+All OpenTelemetry contributors must read the
+[Contributor guide](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)
+and
+[sign the Contributor License Agreement (CLA)](https://docs.linuxfoundation.org/lfx/easycla/contributors)
+.
+
+Pull requests from contributors who haven't signed the CLA fail the automated
+tests. The name and email you provide must match those found in your
+`git config`, and your git name and email must match those used for the CNCF
+CLA.
+
+## Contribute new content
+
+```mermaid
+flowchart LR
+    subgraph first[How to contribute]
+    direction TB
+       T[ ] -.-
+       A[Sign the CNCF CLA] --> D[Write docs in markdown<br>and build site with Hugo]
+       D[Write docs in markdown<br>and build site with Hugo] --- E[Push source to GitHub]
+       E --- G[Open a pull request]
```

Comment:
Signing the CLA is 95% of the time something that happens AFTER the pull request has been opened. Also, I miss the piece around forking the repo and pushing to the fork first and then opening the PR